cmd/fyne/internal/mobile/build_iosapp.go

```diff
@@ -153,6 +153,12 @@ func goIOSBuild(pkg *packages.Package, bundleID string, archs []string,
 			return nil, err
 		}
 	}
+
+	// Use codesign to sign the application so that it can be used in iOS simulator.
```

Comment:
Isn't this handled in the "fyne package" command?

Reply:
I figured. The `fyne package` was for codesign in a real device, if we are installing to a simulator, then the certificate should be removed.

See changes in a new commit.